beacon-chain/core/blocks/block_operations.go

```diff
@@ -1024,6 +1082,9 @@ func ProcessVoluntaryExits(
 ) (*stateTrie.BeaconState, error) {
 	exits := body.VoluntaryExits
 	for idx, exit := range exits {
+		if exit == nil || exit.Exit == nil {
+			continue
```

Comment:
@nisdas @terencechain i think it should fail. why should a block contain nil entries?

Reply:
changed to fail